inscriptions/ISic001228.xml

```diff
@@ -192,29 +197,21 @@
     </facsimile>
     <text>
         <body>
-            <div type="edition" xml:space="preserve" subtype="PHI" n="PHI140714">
+            <div type="edition" xml:space="preserve" xml:lang="grc" resp="#GC">
                 <ab>
-                    <lb n="1"/>
-
-
-                    Αἰλίῳ Ἀδριανῷ<lb n="2"/>Ἀντωνείνωι<lb n="3"/>Σεβαστῷ Εὐσεβεῖ<lb n="4"/>
-                    <expan>
-                        <abbr>π</abbr>
-                        <ex>ατρὶ</ex>
-                    </expan>
-                    <expan>
-                        <abbr>π</abbr>
-                        <ex>ατρίδος</ex>
-                    </expan>
-                    .
-                    <lb n="5"/>
+                    <lb n="1"/><name>Αἰλίῳ</name> <name>Ἀδριανῷ</name>
+                    <lb n="2"/><name>Ἀντωνείνωι</name>
+                    <lb n="3"/><name>Σεβαστῷ</name> <name>Εὐσεβεῖ</name>
+                    <lb n="4"/><expan><abbr>π</abbr><ex>ατρὶ</ex></expan><expan><abbr>π</abbr><ex>ατρίδος</ex></expan>
                 </ab>
             </div>
             <div type="apparatus">
-                <p><!--external apparatus criticus (if applicable)--></p>
+                <listApp>
+                    <app><note>Text of IG</note></app>
+                </listApp>
             </div>
             <div type="translation">
-                <p><!--translation(s) - add @xml:lang for 'en' or 'it' to div--></p>
+                <p>To Aelius Adrianus Antoninus Sebatus Eusebius, father of the country.</p>
```

Comment:
NB Sebastos Eusebes, which in turn can be rendered, surely, as Augustus Pius (this being a version of https://en.wikipedia.org/wiki/Antoninus_Pius)